.all-contributorsrc

```diff
@@ -0,0 +1,15 @@
+{
+  "projectName": "MultiEmail-backend",
+  "projectOwner": "(Ayush Chugh)",
```

Comment:
I am not only owner of the project so change `projectOwner` field to `Multi Email`